### PR TITLE
Upgrade to v3.60.0 of the AWS Terraform Provider

### PR DESCRIPTION
**Pull request by stack72**
Fixes: #1638﻿

### CODE REVIEW

File: provider/resources.go [hunk 1]
```diff
@@ -1455,6 +1465,7 @@ func Provider() tfbridge.ProviderInfo {
 			"aws_fsx_lustre_file_system":  {Tok: awsResource(fsxMod, "LustreFileSystem")},
 			"aws_fsx_windows_file_system": {Tok: awsResource(fsxMod, "WindowsFileSystem")},
 			"aws_fsx_backup":              {Tok: awsResource(fsxMod, "Backup")},
+			"aws_fsx_ontap_file_system":   {Tok: awsResource(fsxMod, "OnTapFilesystem")},
```
Comment (mikhailshilkov) [inline provider/resources.go:1468]: Should it be `OntapFileSystem` : similar to `WindowsFileSystem` above and I think "ONTAP" is a single acronym or brand name (I'm able to find `OntapVolumeConfiguration` in AWS C# SDK?

**Review by mikhailshilkov — COMMENTED** [review]
One naming suggestion, LGTM otherwise

File: provider/resources.go (hunk 1 above)
Comment (stack72) [reply provider/resources.go:1468]: You are correct here! I will update it - good suggestion

Comment (stack72) [reply provider/resources.go]: Updated now @mikhailshilkov

**Review by mikhailshilkov — APPROVED** [review]

**Pull request merged**